include/parallelzone/runtime/runtime_view.hpp

```diff
@@ -490,6 +493,17 @@ class RuntimeView {
     // -- Utility methods
     // -------------------------------------------------------------------------
 
+    /** @brief Adds callback function to call when destructed.
+     *
+     *  Adds functions to a stack of callback functions that will be called
+     *  upone the destruction of *this.
+     *
+     *  @param[in] cb_func The callback function to add to the stack
+     *
+     *  @throw None No throw guarantee.
```

Comment:
```suggestion
     *  @throw std::bad_alloc if there is problem adding the function
     *                        to the stack. Strong throw guarantee.
```
I'm assuming it's strong throw. Does it also throw if there's no PIMPL?